src/components/projects/project-edit-form/project-edit-container/project-edit-details/project-form-detail/project-form-detail.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+import { useContext } from 'react';
+import { Project } from 'utils';
+import { EditModeContext } from '../../project-edit-container';
+import {
+  weeksPipe,
+  wbsPipe,
+  endDatePipe,
+  fullNamePipe,
+  listPipe
+} from '../../../../../../shared/pipes';
+import { Form } from 'react-bootstrap';
+
+interface ProjectFormDetailProps {
+  type: string;
+  isReadonly: boolean;
+  details: Project;
+}
+
+class Modifier {
+  title: string;
+  inputType: string;
+  value: any;
+  constructor(title: string, inputType: string, value: any) {
+    this.title = title;
+    this.inputType = inputType;
+    this.value = value;
+  }
+}
+
+const ProjectFormDetail: React.FC<ProjectFormDetailProps> = ({ type, isReadonly, details }) => {
+  const editMode = useContext(EditModeContext);
+  const workPkg = details.workPackages[0];
```

Comment:
lint check

Reply:
the key is to also include "eslint", the name of our linter, in the google search. Googling "eslint use array destructuring" yields [this article](https://eslint.org/docs/rules/prefer-destructuring) as the result which describes how the rule is applied. using this kind of syntax is discouraged, so you want to ask yourself what you're actually trying to do by accessing the first element of the array. can you do it a different way? or would you prefer to simply suppress the lint error and move on?